docs/en/interfaces/formats.md

```diff
@@ -307,24 +307,68 @@ See also the `JSONEachRow` format.
 
 ## JSONEachRow {#jsoneachrow}
 
-Outputs data as separate JSON objects for each row (newline delimited JSON).
+You can use it both in the `SELECT` and `INSERT` queries. When you select the data in this format, ClickHouse returns each row as a JSON object. When inserting the data, you should provide each row as a JSON object.
```

Comment:
actually, previous version of this description was more precise and clear...

* it's important that objects are separate (not one large object/array for all rows) and even newline is not necessary, it's still gave the name for this format and worth mentioning
* there's a table in the beginning that shows which formats are supported for SELECT/INSERT, why repeat this here?